ood-portal-generator/spec/application_spec.rb

```diff
@@ -159,6 +159,7 @@
       it 'generates full dex configs with SSL' do
         allow(described_class).to receive(:context).and_return({
           servername: 'example.com',
+          proxy_server: 'example-proxy.com',
```

Comment:
I would almost prefer a new test section that is like `generates full dex configs with SSL using a proxy` that way we still have good test coverage for the more common case of Dex with SSL without using a proxy.

Reply:
I think that's a good idea Trey. I went ahead and broke them apart. Hopefully that looks better.

Reply:
Travis is going to make a new fixture file for this now, but at some point we should refactor all these fixture files or come up with some better strategy to test all the cases as having a new fixture file for every single option just isn't tenable.

Reply:
Seems like now that have a new fixture, we should revert this because it's the more common case.

Reply:
Sorry! I see now that this is a fully new test case and not an update to an existing one.